n/node-sass/build_info.json

```diff
@@ -0,0 +1,14 @@
+{
+    "package_name":"node-sass",
+    "github_url":"https://github.com/sass/node-sass",
+    "package_dir":"n/node-sass/",
+    "version":"v8.0.0",
+    "docker_build":false,
```

Comment:
@shubham-garud  Can you please add default_branch and build_script entries in default section here.
```
    "package_name":"node-sass",
    "github_url":"https://github.com/sass/node-sass",
    "package_dir":"n/node-sass/",
    "version":"v8.0.0",
    "docker_build":false,
    "build_script":node-sass_v8.0.0_ubi_8.7.sh,
    "default_branch":master,
```